src/map/status.c

```diff
@@ -2259,6 +2264,8 @@ int status_calc_pc_(struct map_session_data* sd, enum e_status_calc_opt opt) {
 	//Give them all modes except these (useful for clones)
 	bstatus->mode = MD_MASK&~(MD_BOSS|MD_PLANT|MD_DETECTOR|MD_ANGRY|MD_TARGETWEAK);
 
+	status->calc_pc_additional(sd,opt);
```

Comment:
refering to my comment, this would be a if( status->calc_pc_additional ) status->calc_pc_additional(sd,opt);